docs/src/modules/components/AppSearch.js

```diff
@@ -253,7 +253,11 @@ export default function AppSearch() {
             transformItems={(items) => {
               return items.map((item) => {
                 const parseUrl = document.createElement('a');
-                parseUrl.href = item.url;
+                if (['lvl2', 'lvl3'].includes(item.type)) {
+                  parseUrl.href = item.url.replace('#heading-', '#');
```

Comment:
Why? What are some specific examples where this is important?

Reply:
My purpose is to fix this tiny experience. In the clip, the title is not appear below the Header.

https://user-images.githubusercontent.com/18292247/135991437-9340ac73-5e52-4699-b011-b7a3cac762cf.mov

It is because the hash has extra `heading-`. This PR remove the extra `heading-` from the link.
<img width="586" alt="Screen Shot 2564-10-05 at 15 52 12" src="https://user-images.githubusercontent.com/18292247/135991883-1d260a18-864c-4fe1-8f78-8661bf2c8422.png">